packages/bot/src/scratch/hooks/data_category.js

```diff
@@ -44,16 +44,6 @@ Blockly.DataCategory.search = function(variableModelList){
             xmlList.push(setBlock);
         }
 
-        // Create 'Change `var` by `1`'-block
-        if (Blockly.Blocks.math_change) {
-            const gap = Blockly.Blocks.variables_get ? 20 : 8;
-            const changeBlockText = `<xml><block type="math_change" gap="${gap}">${generateVariableFieldXmlString(
-                firstVariable
-            )}<value name="DELTA"><shadow type="math_number"><field name="NUM">1</field></shadow></value></block></xml>`;
-            const changeBlock = Blockly.Xml.textToDom(changeBlockText).firstChild;
-            xmlList.push(changeBlock);
-        }
-
```

Comment:
Can you comment this instead of delete it? In the future, if we want to put this block back to `variable` category we might need these code.

Reply:
done